cirq/google/engine/engine_sampler.py

```diff
@@ -64,3 +64,34 @@ def run_sweep(
     @property
     def engine(self) -> 'cirq.google.Engine':
         return self._engine
+
+
+def get_engine_sampler(processor_id: str, gate_set_name: str,
+                       project_id: Optional[str] = None) \
+        -> 'cirq.google.QuantumEngineSampler':
+    """Get an EngineSampler assuming some sensible defaults.
+
+    This uses the environment variable GOOGLE_GLOUD_PROJECT for the Engine
+    project_id, unless set explicitly. We assume you are using ProtoVersion.V2.
+
+    Args:
+        processor_id: Engine processor ID (from cloud console)
```

Comment:
or from `Engine.list_processors`.  Also nit: period to be consist with other args.

Reply:
done